lib/disposable.dart

```diff
@@ -181,4 +193,10 @@ abstract class Disposable implements _Disposable {
     _didDispose.complete();
     return null;
   }
+
+  void _throwIfNull(dynamic subscription, String name) {
```

Comment:
Can we consider making this a utility method exposed form this library? I could see uses for this beyond a 'subscription'.